src/pages/admin/AdminCustomerInstitutions.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { getAllCustomerInstitutions } from '../../api/customerInstitutionsApi';
+import Card from '../../components/Card';
+import { Button, CircularProgress } from '@material-ui/core';
+import InstitutionList from './InstitutionList';
+import { Link as RouterLink } from 'react-router-dom';
+import { CustomerInstitution } from '../../types/customerInstitution.types';
+import { setNotification } from '../../redux/actions/notificationActions';
+import { NotificationVariant } from '../../types/notification.types';
+import { StyledProgressWrapper, StyledRightAlignedButtonWrapper } from '../../components/styled/Wrappers';
+import { PageHeader } from '../../components/PageHeader';
+
```

Comment:
denne fila er egentlig veldig gammel, men blir "ny" fordi den er renamed (fjerne "Page" postfix).
Kan godt flytte datafetching her til egen hook i samme slenge, men vil ikke gjøre for mye ekstra i denne runden heller. Det viktigste er at ting virker igjen

Reply:
ja, enig i at du ikke trenger å gjøre dette nå